eth/api_backend.go

```diff
@@ -533,11 +533,14 @@ func (b *EthApiBackend) GetEpochDuration() *big.Int {
 func (b *EthApiBackend) GetMasternodesCap(checkpoint uint64) map[common.Address]*big.Int {
 	checkpointBlock := b.eth.blockchain.GetBlockByNumber(checkpoint)
 	state, err := b.eth.blockchain.StateAt(checkpointBlock.Root())
-
 	if err != nil {
 		fmt.Println("ERROR Trying to getting state at", checkpoint, " Error ", err)
 		return nil
 	}
+	if state == nil {
+		fmt.Println("nil state in GetMasternodesCap at checkpoint", checkpoint)
```

Comment:
can you help me use `log.Error` instead of `fmt.Println` please, together with above err condition.